sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FileStreamSink.scala

```diff
@@ -55,6 +54,13 @@ object FileStreamSink extends Logging {
     }
   }
 
+  def getMetadataLogPath(path: Path, hadoopConf: Configuration, sqlConf: SQLConf): Path = {
+    val metadataDir = new Path(path, FileStreamSink.metadataDir)
+    val fs = metadataDir.getFileSystem(hadoopConf)
+    FileStreamSink.checkEscapedMetadataPath(fs, metadataDir, sqlConf)
```

Comment:
this is new code. So this PR fixes a bug instead of just fixing a test?

Reply:
Oh sorry I should have explained before. That's to avoid test to have same code what FileStreamSink does, so that we don't break the test when we somehow change it.